lib/lists/list_info.dart

```diff
@@ -57,7 +59,123 @@ class ListInfo extends ConsumerWidget {
+                                                    child: Row(
+                                                      children: <Widget>[
+                                                        Text("url: ",
+                                                            style: TextStyle(
+                                                                fontWeight:
+                                                                    FontWeight
+                                                                        .bold)),
+                                                        Expanded(
+                                                          child: Container(
+                                                            child: Text(
+                                                                generateUrl(
+                                                                    entityId)),
+                                                          ),
+                                                        ),
+                                                        ElevatedButton(
+                                                          child: Text("copy"),
+                                                          onPressed: () async =>
+                                                              {
+                                                            await Clipboard.setData(
+                                                                ClipboardData(
+                                                                    text: generateUrl(
+                                                                        entityId)))
+                                                          },
+                                                        ),
+                                                      ],
+                                                    )),
+                                              ],
+                                            )
+                                          ],
+                                        ),
+                                      )),
+                                ),
+                                actions: [
+                                  ElevatedButton(
+                                    onPressed: () {
+                                      Navigator.of(context).pop();
+                                    },
+                                    child: const Text('Done'),
+                                  )
+                                ],
+                              );
+                            });
+                      },
+                    ))
+                  ],
+                )
               ]));
 }
+
+String generateCurlUrl(String action, String resource) {
+  return "curl -X ${action} -H 'Content-Type:application/json' '${dotenv.env['API_URL']}/GetSanctionListEntities?list=${resource}' ";
```

Comment:
maybe API function names should also be in the variable? Or a separate variable?